applications/qserv-kafka/templates/frontend-deployment.yaml

```
@@ -61,6 +61,10 @@ spec:
                 secretKeyRef:
                   name: "qserv-kafka"
                   key: "redis-password"
+            {{- if .Values.frontend.debug.enabled | and .Values.frontend.debug.disablePymalloc }}
```

```suggestion
            {{- if (and .Values.frontend.debug.enabled .Values.frontend.debug.disablePymalloc) }}
```
seems a bit easier to read to me? Might be a matter of taste.